osdep/io.h

```diff
@@ -59,6 +59,15 @@ char *mp_to_utf8(void *talloc_ctx, const wchar_t *s);
 #include <io.h>
 #endif
 
+#ifdef _WIN32
+#include <windows.h>
+#define dlopen(filename, flags) LoadLibrary(TEXT(filename))
```

Comment:
I'll drop it. I can't test on windows, so going further down this path won't end well for me :-)